.eslintrc.js

```diff
@@ -238,7 +238,6 @@ module.exports = {
                 'jsdoc/no-types': 'error',
                 'rulesdir/no-default-props': 'error',
                 'import/no-extraneous-dependencies': 'off',
-                'rulesdir/prefer-underscore-method': 'off',
```

Comment:
@blazejkustra We got lint errors when I removed this. Do we want to update `eslint-config-expensify` to remove this rule or keep turning this rule off?

Reply:
I feel like we should `eslint-config-expensify`.

Reply:
@mountiny can you confirm?

_I will update this tomorrow_

Reply:
I have turned this off and added back `npm install underscore` in `CIGitLogicTest` for now since we don't have any answer yet. I think you can start reviewing it. @hungvu193

Reply:
Btw, I ask for the help in [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1713523057689269)

Reply:
Sure. I'll review in a while